src/app/screens/SignUp/components/Form/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import styles from './styles.module.scss';
+
+export default function Form() {
+  const { register, handleSubmit } = useForm();
+  const onSubmit = data => {
+    console.log(data);
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} className={`column ${styles.form}`}>
+      <label className={styles.label}> Nombre </label>
```

Comment:
in your training board you could see an internationalization card, use i18n in its default to define texts, and labels